lib/models/visualRules.js

```diff
@@ -38,6 +38,13 @@ var util = require('util'),
     },
     errors = {};
 
+function errorOperator(op) {
```

Comment:
Sorry for this post-LGTM comment, but I have just realized...

Do we need entry at CHANGES_NEXT_RELESE for these changes? Probably not, due that are small and internal ones, but just to let you know...